draft-ietf-quic-transport.md

```diff
@@ -674,6 +689,24 @@ packet protection in detail.  After decryption, the plaintext consists of a
 sequence of frames, as described in {{frames}}.
 
 
+## Compound Packets {#packet-compound}
+
+A sender can combine multiple QUIC packets (typically a Cryptographic Handshake
+packet and a Protected packet) into one UDP datagram.  This can reduce the
+number of UDP datagrams required to be emitted when application data can be sent
+during the handshake.  A packet with a short header does not include a length,
+so it has to be the last packet included in a UDP datagram.
+
+The sender MUST NOT combine QUIC packets belonging to different QUIC
+connections into a single UDP datagram.
+
```

Comment:
What about packets in the gray zone such as version negotiation and resets?

Reply:
Yes, I will address those in a follow-up.